services/src/main/java/org/exoplatform/news/NewsServiceImpl.java

```diff
@@ -671,11 +702,15 @@ public void deleteNews(String newsId) throws Exception {
   }
 
   @Override
-  public News convertNodeToNews(Node node) throws Exception {
+  public News convertNodeToNews(Node node, boolean editMode) throws Exception {
     if (node == null) {
       return null;
     }
-
+    if (!editMode && node.getProperty(CURRENT_STATE).getString().equals(DRAFT) && node.hasProperty(LIVE_REVISION_PROP)) {
+      String nodeVersionUUID = node.getProperty(LIVE_REVISION_PROP).getString();
```

Comment:
```suggestion
      String versionNodeUUID = node.getProperty(LIVE_REVISION_PROP).getString();
```
Change the name of the variable also in deleteNews method also